<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SqlQuery</class>
 <widget class="QMainWindow" name="SqlQuery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SqlQuery</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1,1">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" colspan="2">
        <widget class="QTableView" name="tableView"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="4" column="0" colspan="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Name</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Sex</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="9" column="2" colspan="2">
        <widget class="QSpinBox" name="spinBoxAge"/>
       </item>
       <item row="6" column="2" colspan="2">
        <widget class="QComboBox" name="comboBoxSex"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>ID</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Birth</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="2">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Age</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="10" column="2" colspan="2">
        <widget class="QDoubleSpinBox" name="doubleSpinBoxSalary"/>
       </item>
       <item row="4" column="2" colspan="2">
        <widget class="QLineEdit" name="lineEditName"/>
       </item>
       <item row="8" column="0" colspan="2">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Country</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="2" colspan="2">
        <widget class="QSpinBox" name="spinBoxID"/>
       </item>
       <item row="8" column="2" colspan="2">
        <widget class="QLineEdit" name="lineEditCountry"/>
       </item>
       <item row="10" column="0" colspan="2">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Salary</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="7" column="2" colspan="2">
        <widget class="QDateEdit" name="dateEdit"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="LabImage">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>982</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionFirst"/>
   <addaction name="actionBack"/>
   <addaction name="actionNext"/>
   <addaction name="actionLast"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/open3.bmp</normaloff>:/icons/open3.bmp</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionFirst">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/308.bmp</normaloff>:/icons/308.bmp</iconset>
   </property>
   <property name="text">
    <string>First</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/310.bmp</normaloff>:/icons/310.bmp</iconset>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/312.bmp</normaloff>:/icons/312.bmp</iconset>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </action>
  <action name="actionLast">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/314.bmp</normaloff>:/icons/314.bmp</iconset>
   </property>
   <property name="text">
    <string>Last</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/132.bmp</normaloff>:/icons/132.bmp</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>SqlQuery</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>490</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
